shared/attributes.asciidoc

```diff
@@ -60,6 +59,10 @@
 :curator-ref:          https://www.elastic.co/guide/en/elasticsearch/client/curator/{branch}
 :curator-ref-current:  https://www.elastic.co/guide/en/elasticsearch/client/curator/current
 :infra-guide:          https://www.elastic.co/guide/en/infrastructure/guide/{branch}
+:metrics-ref:          https://www.elastic.co/guide/en/metrics/{branch}
+:metrics-guide:        https://www.elastic.co/guide/en/metrics/guide/{branch}
+:logs-ref:             https://www.elastic.co/guide/en/logs/{branch}
+:logs-guide:           https://www.elastic.co/guide/en/logs/guide/{branch}
```

Comment:
Any reason why you want to include these changes in this PR? If you break these changes into a separate PR it would allow you to merge the Kibana and Beats changes prior to splitting the book. That would make the process much simpler than trying to merge 4 PRs at the same time.

Reply:
Because that's the way it came out. These definitions are needed to resolve broken links else where in the docs. I'm really reluctant to spend any more time unpicking and reworking this any further this unless there is a clear benefit. As far as I can tell, as implemented, this builds across all the affected repos. Any more fiddling is probably going to break things.

Reply:
Sounds good